crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -269,8 +267,10 @@ impl FromWorld for MeshPipeline {
             Res<RenderDevice>,
             Res<DefaultImageSampler>,
             Res<RenderQueue>,
+            Res<RenderTextureFormat>,
         )> = SystemState::new(world);
-        let (render_device, default_sampler, render_queue) = system_state.get_mut(world);
+        let (render_device, default_sampler, render_queue, firstly_available_texture_format) =
```

Comment:
Nit: I think gramattically this should be `first_available_texture_format)`

Reply:
Done. :)